OpenRA.Mods.Common/Traits/CombatDebugOverlay.cs

```diff
@@ -51,8 +51,20 @@ public void RenderAfterWorld(WorldRenderer wr, Actor self)
 			var iz = 1 / wr.Viewport.Zoom;
 
 			if (healthInfo != null)
+			{
 				healthInfo.Shape.DrawCombatOverlay(wr, wcr, self);
 
+				var targetZc = Color.Magenta;
+				var tx = new WVec(96, 0, 0);
```

Comment:
> What is this 96 value and why 96?

Offset relative to TargetingCenterPosition, used to define the start and end points of the lines for drawing the cross overlay.

> Can this logically be customizable?

Yes. But nearly all other debug overlays we have aren't customizable, either.